roles/template_service_broker/tasks/remove.yml

```diff
@@ -0,0 +1,25 @@
+---
+- local_action: command mktemp -d /tmp/tsb-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+  become: no
+
+- name: Pull down template
+  command: >
+    wget {{ __tsb_files_location }}{{ __tsb_template_file }} -P {{ mktemp.stdout }}
+
+- name: Pull down config
+  command: >
+    wget {{ __tsb_files_location }}{{ __tsb_config_file }} -P {{ mktemp.stdout }}
+
+- name: Delete template file
+  command: >
+    kubectl delete -f {{ mktemp.stdout }}/{{ __tsb_template_file }}
```

Comment:
kubectl versus oc in the next task, is that intentional?

Reply:
@deads2k delete on a template will delete all the resources defined in the template?

Reply:
> @deads2k delete on a template will delete all the resources defined in the template?

You need to process the template, then pipe that to delete.  You probably want to delete the entire namespace (after doing this) just in case.

Reply:
I was going to suggest just nuking the namespace.

Reply:
> I was going to suggest just nuking the namespace.

both are useful.  Some resources which are created are cluster scoped.  ClusterRoles for instance.